Tw 171 convert external resources to kb (#1841)
temporalio/documentation: kb/how-to-explain-temporal.md

---
slug: how-to-explain-temporal
title: How to explain Temporal
tags:
  - resources
date: 2022-12-12T00:00:00Z
---

There are many ways to introduce and teach Temporal based on your background.
Temporal doesn't have a monopoly on explaining Temporal.

<!-- truncate -->

Here, we list great blogposts and videos by Temporal staff and users that might be helpful in increasing your understanding or convincing your team.

> If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/additional-resources.md) so that others can find it!

Reviewer: ```suggestion
If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/how-to-explain-temporal.md) so that others can find it!
```

<!-- truncate -->

## Press announcements

There have been many high-level descriptions of Temporal from our Series A and B PR. This may be helpful for non-technical or technical-adjacent audiences:

- **Series B (2022)** Linked in [our Series B press roundup](https://temporal.io/blog/series-b-press)
- **Series A (2020)**: Linked in [our Series A blogpost](https://temporal.io/blog/funding-announcement)

## How Temporal works

> > See also: [Concepts ➡️ What is Temporal](/temporal)

Reviewer: ```suggestion
See the Concepts topic about [Temporal](/temporal).
```

  - (Go) [Creating a custom Temporal data convertor for encryption](https://nightfall.ai/creating-a-custom-temporal-data-convertor-for-encryption) by Chris Cowell and Rohan Sathe
- Talks
  - (TypeScript) [Temporal - React for the Backend](https://changelog.com/jsparty/208) on JSParty (57 min listen)
  - (PHP) [Fault tolerant workflow orchestration on PHP](https://www.youtube.com/watch?v=pdxHkIqX62A) at PHPKonf 2021 by Anton Titov (37 min talk)
- Other Languages
  - (Español) [Construye aplicaciones invencibles con Temporal](https://sistecma.github.io/2021/02/04/aplicaciones-invencibles-con-temporal.html) y [Aplicando el patrón SAGA en tus microservicios construidos](https://sistecma.github.io/2021/03/04/aplicando-saga-en-microservicios-con-temporal.html) by Hernán Moreno
  - (Türkçe) [Nedir Bu Temporal.IO?](https://alameddinc.medium.com/nedir-bu-temporal-io-680f3b242136) by Alameddin Çelik
  - (русский) [PHP Russia talks](https://legacy-documentation-sdks.temporal.io/php/introduction/#resources) by Anton Titov
  - (中文) [Temporal 简介](https://www.yuque.com/abser/txixor/oz3yav) by Abser Ari 杨鼎睿

> - _Submit yours [here](https://github.com/temporalio/documentation/edit/master/kb/additional-resources.md)!_

Reviewer: ```suggestion
- _Submit yours [here](https://github.com/temporalio/documentation/edit/master/kb/how-to-explain-temporal.md)!_
```

## Community distros

- [Airbyte](https://github.com/airbytehq/airbyte) - [ELT data pipeline platform (see talk)](https://www.youtube.com/watch?v=K25Bt5asd8I)
- [Roadrunner](https://github.com/roadrunner-server/roadrunner) - PHP application server
- [Coinbase/temporal-ruby](https://github.com/coinbase/temporal-ruby) - Ruby SDK with Rails example
- [Nunchi Blacksmith](https://www.nunchi.studio/blacksmith/start/onboarding/how) - ETL data platform

> ## Misc

Reviewer: ```suggestion
## Miscellaneous
```

_Everything that doesn't fit above goes here._

> If you can't find what you need, don't hesitate to ask on the Temporal Slack, or search in the Forums where there are common questions like:

Reviewer: ```suggestion
If you can't find what you need, don't hesitate to ask on the Temporal Slack workspace or search in the [Temporal Community Forum](https://community.temporal.io/), where you can find answers to common questions like the following:
```
